libs/CMakeLists.txt

```diff
@@ -49,6 +46,15 @@ set(HPX_LIBS
   CACHE INTERNAL "list of HPX modules" FORCE
 )
 
+if(NOT HPX_WITH_EXPERIMENTAL_LOCAL_RUNTIME)
+  set(HPX_LIBS ${HPX_LIBS}
+    collectives
+    resiliency
+    segmented_algorithms
+    CACHE INTERNAL "list of HPX modules" FORCE
+  )
+endif()
```

Comment:
I think resiliency does not rely on distributed capabilities. Everything is purely local.

Reply:
Makes sense. Nice!